app/views/job_invocations/_form.html.erb

```diff
@@ -65,6 +65,9 @@
                       <% end %>
                     <% end %>
                   <% end %>
+                  <% if job_template.effective_user.overridable? %>
+                    <%= text_f job_template_fields, :effective_user, :label => _('Effective user') %>
```

Comment:
Something like this? https://github.com/theforeman/foreman_remote_execution/commit/28a786d44f92ffaf80bce7092654b5ba7f8b2bb5